attack.go

```diff
@@ -177,6 +177,9 @@ func (h headers) String() string {
 	return buf.String()
 }
 
+// Set implements the flag.Value interface. It's used to set an
+// arbitrary number of headers that have the same key which are
+// valid HTTP semantics.
```

Comment:
Sorry, I'd rather not have this comment here. Maybe: `Set implements the flag.Value interface for a map of HTTP Headers.`